hw/ip/kmac/data/kmac.hjson

```diff
@@ -192,20 +192,26 @@
           name: "msg_endianness"
           desc: '''Message Endianness.
 
-                If 1, Convert TL-UL write data[31:0] to {d[7:0], d[15:8], ...}
-
-                0: Little-endian. Keep input value as it is
-                1: Big-endian. Convert incoming value
+                If 1 then each individual multi-byte value, regardless of its
```

Comment:
question - maybe @eunchan can confirm, but is it worth specifying that the endian-swapping is done at a TLUL-word granularity? It looks like this is more or less stated in the below section for `state_endianness`, but IMO it's good to be even more explicit for both CSR fields as this can definitely cause some confusion.

Reply:
Yeah. I think it is clearer to mention that in this field.

Reply:
How about a sentence at the end just saying that explicitly:

`From a hardware perspective byte swaps are performed on a TL-UL word granularity.`

Does that make sense? I don't think we should assume that software developers know what TL-UL is so this sentence would be more for hardware developers.

Reply:
I think that would be fine, thanks!

Reply:
Thanks! I've added that sentence to the message endianness flags.